website/integrations/services/engomo/index.md

```diff
@@ -0,0 +1,81 @@
+
+## What is Engomo
+
+> Engomo is an enterprise application to create apps for smartphones/tablets based on Android, iOS or iPadOS.
+> -- https://engomo.com/
+>
+> This guide explains how to setup a Engomo to use authentik as OAUTH provider for the application login on the smartphone/tablet and login to the admin WebGUI (composer).
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `engomo.company` is the FQDN of the Engomo install.
+-   `authentik.company` is the FQDN of the authentik install.
+-   `engomo.mapping` is the name of the Scope Mapping.
+-   `ak.cert` = The authentik self-signed certificate you use for the service provider.
+
+## Custom Property Mapping
+
+Create a new Scope Mapping under the Customization settings (Property Mapping).
+
+-   `engomo.mapping` is the value for the Name.
+-   `profile` is the value for the Scope name.
+-   `return {"preferred_username": request.user.email,}` is the value for the Expression.
+
+Create an application and OAuth2/OpenID Provider in authentik. Create a OAuth2/OpenID Provider with the following parameters:
+
+Provider:
+
+-   Name: `SP-engomo`
+-   Client type: `Public`    
+-   Redirect URIs/Origins (RegEx) #1: `https://engomo.company/auth` (for composer)
+-   Redirect URIs/Origins (RegEx) #2: `com.engomo.engomo://callback/` (for app)
+-   Signing Key: `ak.cert`
+-   Scopes: `authentik default OAuth Mapping: OpenID 'email'`
+-   Scopes: `authentik default OAuth Mapping: OpenID 'offline_access'`
+-   Scopes: `authentik default OAuth Mapping: OpenID 'openid'`
+-   Scopes: `engomo.mapping`
+-   Note the Client ID and Client Secret.
+
+> [!IMPORTANT]
+> Redirect URIs => write the values line by line.
+
+Lave the rest as default values. You can of course adjust durations.
+
+Application:
+
+-   Name: `Engomo`
+-   Slug: `engomo`
+-   Launch URL: `https://engomo.company/`
+
+## Engomo Configuration
+
+Navigate to `https://engomo.company/composer` and login with your admin credentials.
+
+-   Select `Server`.
+-   Select `Authentication`.
+-   Add a new authentcation method by clicking on the plus icon on the right.
+-   Name: `authentik`
+-   Type: `OpenID Connect`
+-   Click on Create.
+-   Set the `Issuer` to the authentik FQDN `https://authentik.company/application/o/engomo`.
+-   Set the `Client ID` to the Client ID from the SP-engomo provider inside authentik.
+-   Set the `Client Secret` to the Client Secret from the SP-engomo provider inside authentik.
+
+Leave the rest as default.
+
+## Engomo User Creation
+
+Engomo doesn't create users automatically on signing in. So you have to do it manually right now.
+Navigate to `https://engomo.company/composer` and login with your admin credentials.
+
+- Select `Users & Devices`.
+- Click the plus button next in the Users section.
+- Select `authentik` as the Authenticator in the dropdown.
+- Create your user by typing in the email as the Username used in authentik.
```

Comment:
Does this mean "Create your user by typing in the email address that you defined as the Username in authentik." ?

I'm not sure I follow it, the way it is written.

Also, if there are there any validation steps that one can take, or next steps, it would be good to have a wrap up sentence at end. Maybe something like "Now your engomo apps are configured to use authentik for login authentication." or some such.

Reply:
Thais Right. The admin ahpuld cremte Users with their corresponding email from authentik to be able to use it and leg the Users Log in.

Will do those steps later. Learned more about english 😎 First I have to figure out on how to run the scripts inside WebGUI of github.

Useally I only fork a repo and then edit it in the browser :) there are 1 or 2 questions I have to ask the engomo Support before I can answer here or accept a Commit or change.

Capital and small letters are made with my Smartphone in this answer, sry for that 😩

Reply:
Yeah, you can't run the linters from the GitHub web UI ... Here are other options to get the `prettier` lint run:
1. I would encourage you to fork and clone our repo to your local environment, and then use a terminal. It's a great way to practice the basics of Git and become comfortable with command lines. I personally think CLIs are way more fun than UIs, but don't let my bias sway you, haha. Or, if you fork and clone to a local repo, you can use one of the many third-party git management tools with a UI.
2. I can do it, and push the fix commit to your PR branch

We appreciate this contribution! Let me know how you'd like to proceed.

Reply:
I wrote you an email to get this done in any way, want to learn this stuff :)
Btw: Thank you very much for reviewing my PR! You did awesome job on my FortiGate things months ago! I appreciate it.